tox.ini

```diff
@@ -14,7 +14,7 @@ DJANGO =
     master: djangomaster
 
 [testenv]
-commands = ./runtests.py --fast --coverage {posargs}
+commands = python -W error::DeprecationWarning -W error::PendingDeprecationWarning runtests.py --fast --coverage {posargs}
```

Comment:
Btw the reason to do this with `python -W` rather than through e.g. pytest's warning system is to catch any import-time warnings that happen before pytest even starts running.